.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,8 +9,7 @@ assignees: ''
 <!-- Thanks for your interest in the project. We appreciate bugs filed and PRs submitted! -->
 
 <!--
-- For questions related to using the library, please visit a support community
-  instead of filing an issue on GitHub: https://spectrum.chat/testing-library
+- For questions related to using the library, please [join the Discord server](https://testing-library.com/discord) instead of filing an issue on GitHub.
```

Comment:
Note that this wouldn't get rendered since it's in a comment. Maybe just put the link in parentheses.
```suggestion
- For questions related to using the library, please join the Discord server (https://testing-library.com/discord) instead of filing an issue on GitHub.
```

Reply:
right! thanks